src/d/a/d_a_base.cpp

```diff
@@ -562,9 +656,19 @@ void dAcBase_c::doInteraction(s32 param) {
     }
 }
 
-// Only called by dPlayer::dig and that function fails to decomp in ghidra?
+// Only called by dPlayer::dig
+// Rounds angle to nearest 90 deg?
 // 8002dc20
-void dAcBase_c::FUN_8002dc20(s16 *, s16 *) {}
+void roundAngleToNearest90(s16 *dst_angle, s16 *src_angle) {
```

Comment:
probably meant to be `dAcBase_c::roundAngleToNearest90` (and make static in header?)